api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -101,6 +106,19 @@ def _open_experiment(
         yield exp
 
 
+def _tables_to_np(
+    tables: Iterator[Tuple[Table, Any]], shape: Tuple[int, int]
+) -> typing.Generator[Tuple[npt.NDArray[Any], Any, int], None, None]:
+    for tbl, indices in tables:
+        row_indices_np = np.array(tbl.columns[0])
```

Comment:
This creates a copy of the Arrow array which is wasteful/unnecessary.

Recommend instead doing a zero-copy conversion:
> row_indices_np = tbl.columns[0].to_numpy()

Same issue for all three columns.

Reply:
👍